clang/lib/Driver/Driver.cpp

```diff
@@ -6380,6 +6380,11 @@ std::string Driver::GetFilePath(StringRef Name, const ToolChain &TC) const {
   if (llvm::sys::fs::exists(Twine(R)))
     return std::string(R);
 
+  SmallString<128> R2(ResourceDir);
+  llvm::sys::path::append(R2, "..", "..", Name);
+  if (llvm::sys::fs::exists(Twine(R2)))
+    return std::string(R2);
```

Comment:
Can you move this in front of `return std::string(Name);`?

Reply:
OK, I just moved it and did a force push. Makes sense to try this as the last thing. Less likely to break something else.